modules/ballerina-builtin/src/main/java/org/ballerinalang/nativeimpl/task/ScheduleAppointment.java

```diff
@@ -0,0 +1,97 @@
+*  You may obtain a copy of the License at
+*
+*    http://www.apache.org/licenses/LICENSE-2.0
+*
+* Unless required by applicable law or agreed to in writing,
+* software distributed under the License is distributed on an
+* "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+* KIND, either express or implied.  See the License for the
+* specific language governing permissions and limitations
+* under the License.
+*/
+package org.ballerinalang.nativeimpl.task;
+
+import org.apache.commons.lang3.StringUtils;
+import org.ballerinalang.bre.Context;
+import org.ballerinalang.model.types.TypeKind;
+import org.ballerinalang.model.values.BFunctionPointer;
+import org.ballerinalang.model.values.BInteger;
+import org.ballerinalang.model.values.BString;
+import org.ballerinalang.model.values.BStruct;
+import org.ballerinalang.model.values.BValue;
+import org.ballerinalang.natives.AbstractNativeFunction;
+import org.ballerinalang.natives.annotations.Argument;
+import org.ballerinalang.natives.annotations.Attribute;
+import org.ballerinalang.natives.annotations.BallerinaAnnotation;
+import org.ballerinalang.natives.annotations.BallerinaFunction;
+import org.ballerinalang.natives.annotations.ReturnType;
+import org.ballerinalang.util.codegen.cpentries.FunctionRefCPEntry;
+
+import java.io.PrintStream;
+
+/**
+ * Native function ballerina.model.task:scheduleAppointment.
+ */
+@BallerinaFunction(
+    packageName = "ballerina.task",
+    functionName = "scheduleAppointment",
+    args = {@Argument(name = "onTrigger", type = TypeKind.ANY),
+            @Argument(name = "onError", type = TypeKind.ANY),
+            @Argument(name = "appointmentScheduler", type = TypeKind.STRUCT, structType = "AppointmentScheduler")},
+    returnType = {@ReturnType(type = TypeKind.INT), @ReturnType(type = TypeKind.ANY)},
+    isPublic = true
+)
+@BallerinaAnnotation(annotationName = "Description", attributes = {@Attribute(name = "value",
+        value = "Schedules the task service with cron expression") })
+@BallerinaAnnotation(annotationName = "Param", attributes = {@Attribute(name = "onTrigger",
+        value = "The schedule function as any type") })
+@BallerinaAnnotation(annotationName = "Param", attributes = {@Attribute(name = "onError",
+        value = "The error function as any type") })
+@BallerinaAnnotation(annotationName = "Param", attributes = {@Attribute(name = "appointmentScheduler",
+        value = "The struct with required attributes") })
+@BallerinaAnnotation(annotationName = "Return", attributes = {@Attribute(name = "int)",
+        value = "The identifier of the task") })
+@BallerinaAnnotation(annotationName = "Return", attributes = {@Attribute(name = "any)",
+        value = "The error which is occurred while scheduling the task") })
+public class ScheduleAppointment extends AbstractNativeFunction {
+
+    public BValue[] execute(Context ctx) {
+        PrintStream out = System.out;
+        out.println("In Schedule Appointment Class!!!!!");
```

Comment:
This should be removed ? we could use logs to print something meaningful ?

Reply:
Removed this line